samples/algorithms/MajoranaQubits.qs

```diff
@@ -0,0 +1,102 @@
+/// # Sample
+/// Majorana Qubits
+///
+/// # Description
+/// In hardware providing majorana qubits, common quantum operations
+/// are implemented using measurements and Pauli corrections. This
+/// sample shows a hypotetical hardware provider exposing some custom
+/// gates to Q# and a small library built on top of it.
+
+namespace Main {
```

Comment:
Rather than have a sample that brings back multiple namespaces in a file, I think it might make more sense to have this as a multifile sample in a folder.